src/js/edu-benefits/1990e/config/form.js

```diff
@@ -116,6 +128,72 @@ const formConfig = {
     educationHistory: {
       title: 'Education History',
       pages: {
+        educationHistory: {
+          path: 'education-history',
+          title: 'Education History',
+          initialData: {
+          },
+          uiSchema: {
+            highSchoolOrGedCompletionDate: date.uiSchema('When did you earn your high school diploma or equivalency certificate?'),
+            'view:educationHistory': {
+              'ui:title': 'Please list any post-high school trainings you have completed.'
```

Comment:
Screenshots for @goldenmeanie to choose from or propose something else:

1. Show title ("Education after high school") which will have <h5> tags, and leave description as non-heading text
![image](https://cloud.githubusercontent.com/assets/25439097/23810167/4eec4516-0585-11e7-8c5f-e2aa9fc37c2d.png)

2. Hide title, and render description ("Please list any post-high school trainings you have completed.") with <h5> tags
![image](https://cloud.githubusercontent.com/assets/25439097/23810234/972ba740-0585-11e7-81b1-b1ae0c05ef8b.png)